pkgs/tools/misc/diffoscope/default.nix

```diff
@@ -82,6 +82,10 @@ python3Packages.buildPythonApplication rec {
 
     # disable formatting tests because they can break on black updates
     "test_code_is_black_clean"
+
+    # fails at 2022-09-30
```

Comment:
This comment is rather useless. This information can easily be looked up with git blame but to know why the tests are failing I would need to revert the commit and start a build.


```
=================================== FAILURES ===================================
_____________________________ test_identification ______________________________

pyc1 = <<class 'diffoscope.comparators.binary.FilesystemFile'> /build/diffoscope-222/tests/data/test1.pyc-renamed>
pyc2 = <<class 'diffoscope.comparators.binary.FilesystemFile'> /build/diffoscope-222/tests/data/test2.pyc-renamed>

    @skip_unless_file_version_is_at_least("5.39")
    def test_identification(pyc1, pyc2):
>       assert isinstance(pyc1, PycFile)
E       AssertionError: assert False
E        +  where False = isinstance(<<class 'diffoscope.comparators.binary.FilesystemFile'> /build/diffoscope-222/tests/data/test1.pyc-renamed>, PycFile)

tests/comparators/test_python.py:37: AssertionError
__________________________________ test_diff ___________________________________

differences = []

    @skip_unless_file_version_is_at_least("5.39")
    @skipif(
        sys.version_info < (3, 8),
        reason="Python 3.7 cannot de-marshal test1.pyc-renamed",
    )
    def test_diff(differences):
        assert_diff_startswith(
>           differences[0],
            "pyc_expected_diff",
        )
E       IndexError: list index out of range

tests/comparators/test_python.py:61: IndexError
```